### ISSUE TITLE
Doesn't build anymore

### ISSUE THREAD

**Issue by kpcyrd** [bluejekyll/enum-as-inner#8]
hello!

I've noticed the current master doesn't build anymore and `dependabot/cargo/proc-macro2-1.0,` resolves most of these issues except this one:

```
error[E0599]: no method named `value` found for type `&syn::Field` in the current scope
   --> src/lib.rs:138:31
    |
138 |             let field = field.value();
    |                               ^^^^^

error[E0599]: no method named `value` found for type `&syn::Field` in the current scope
   --> src/lib.rs:189:31
    |
189 |             let field = field.value();
    |                               ^^^^^

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0599`.
error: Could not compile `enum-as-inner`.

To learn more, run the command again with --verbose.
```

Adding travis to this project could be a good idea.

Thanks!

**Comment by bluejekyll** [comment]
Yes! I’ve been meaning to add Travis. Thanks for the fix!

**Comment by bluejekyll** [comment]
Ok, adding travis in #9, and I believe #10 fixes the build issue, I'll bring in the other changes on top of those.

**Comment by bluejekyll** [comment]
Fixed in #10
